### PR TITLE
Change: confirm button text to create motion

### PR DESCRIPTION
**Pull request by Julianahlk**
## Description

This PR is for changing the Confirm button text to Create Motion when creating a motion. 

New stuff ✨

* Updated text for the confirm button for all forms that create motions. Example:

Screen Shot 2022-06-24 at 4 48 35 PM

Screen Shot 2022-06-24 at 4 48 46 PM

Changes 🏗

- [x] The button text will need to change back to Confirm if the action is forced.
- [x] If the extension Governance ( Reputation Weight ) is disabled the button should show the text confirm.

Example:

Screen Shot 2022-06-24 at 4 54 19 PM

Screen Shot 2022-06-24 at 4 54 32 PM

(Resolves | Contributes to) #3503

### CODE REVIEW

**Comment by arrenv** [comment]
@Julianahlk To request a review from everyone, you can use the `JoinColony/frontend` team
65a0ed13-0611-4b2f-bdf8-de25b0aa8c9e.

**Review by arrenv — APPROVED** [review]
Works great, nice one @Julianahlk!! Congrats on your first PR 🥇

**Comment by Julianahlk** [comment]
Thanks @arrenv I will use that tag next time. Happy it works as intended. Thanks for reviewing it!

**Review by chinins — CHANGES_REQUESTED** [review]
Great work for your first PR. There are 2 things to correct:

- I think that you forgot the `NetworkContractUpgradeDialog`
- In the create motion version of the button the text is not centred as per below. It's a tricky thing to notice and I paid attention because I had to update the button styles in another PR to fix this.
Screenshot 2022-06-27 at 12 08 03

**Comment by rdig** [comment]
> Nice PR! I presume the dialogs that have not been changed (e.g. ManageWhitelistDialogForm (Manage Address Book)) haven't been changed because they don't require a motion to be submitted? Or is every dialog supposed to have this "Create Motion" text?

You are correct, the "missing" dialogs do not create a Motion, as they are basically cheating behind the scene and do not call a contract function, in the same way as the others do

**Comment by rdig** [comment]
@Julianahlk 

> I will center the text. That's something I didn't check because I thought the button would re-size automatically. Thanks for reviewing!

For this you'll have to add local styles that overwrite the `Button` component's default styles. Have a look at other examples and you'll catch on

**Review by rdig — APPROVED** [review]
Other than what Olga found, I'm good with this.

I'm pre-approving, but do please address Olga's concerns before merging

**Comment by Julianahlk** [comment]
> Button

On it! Thank you for the brief explanation.

**Comment by Julianahlk** [comment]
@chinins  @rdig 

I switched the width property to be minWidth so the Create Motion/Confirm button could expand to fit any future size text (localized text). I am not sure if this what you were looking for but I think it is more future proof as the buttons can handle longer text if necessary.

File: src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx [hunk 1]
```diff
@@ -467,10 +467,14 @@ const ManageReputationDialogForm = ({
...
-          text={{ id: 'button.confirm' }}
+          text={
+            values.forceAction || !isVotingExtensionEnabled
+              ? { id: 'button.confirm' }
+              : { id: 'button.createMotion' }
+          }
           loading={isSubmitting}
           disabled={!isValid || inputDisabled}
-          style={{ width: styles.wideButton }}
+          style={{ minWidth: styles.wideButton }}
```
Comment (rdig) [inline src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx:477]: We usually prefer to add this via css styles not via JS.

You can do the same, by overwriting the button `min-width` in the styles file

**Review by rdig — CHANGES_REQUESTED** [review]
Good, but I'd prefer your change be done inside CSS, rather then JS _(components)_

Also, you have a bad merge in your branch
Screenshot from 2022-06-29 01-23-16

If you were attempting to update the branch, you did it the wrong way around, as we use `git rebase` for that.

Let me know if you need help restoring your branch

File: src/modules/core/components/InfoPopover/MemberInfoPopover/UserReputation.tsx [hunk 2]
```diff
@@ -66,51 +69,64 @@ const UserReputation = ({
...
-            values={{ isCurrentUserReputation }}
-          />
-        </p>
+      {isUserReputationLoading ? (
+        <SpinnerLoader
+          appearance={{
+            theme: 'grey',
+            size: 'small',
+          }}
+        />
```
Comment (rdig) [inline src/modules/core/components/InfoPopover/MemberInfoPopover/UserReputation.tsx:78]: How did this changes end up in here ? These are not related to your branch.

Did you cherry-pick them from somewhere? If so, why ?

**Comment by Julianahlk** [comment]
Regarding the bad merge. Im not sure why this happened this time because I did try to use, `git pull --rebase origin master`. It seems like it was a merge of my own branch into my branch. I could use help restoring this.

File: src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx (hunk 1 above)
Comment (Julianahlk) [reply src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx:477]: @rdig  So my goal was to remove the width style and replace it with minWidth. Are you saying that having the width style in the tsx file wasnt the best way to do it to begin with?

Comment (rdig) [reply src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.tsx:477]: Exactly. But now that I'm thinking more about this, in order to expedite this PR, leave it like it is for now.

Let's just fix your branch tommorow, then you can merge it

File: src/modules/core/components/InfoPopover/MemberInfoPopover/UserReputation.tsx (hunk 2 above)
Comment (Julianahlk) [reply src/modules/core/components/InfoPopover/MemberInfoPopover/UserReputation.tsx:78]: It looks like there are these commits from @arrenv earlier today that ended up in my branch, https://github.com/JoinColony/colonyDapp/pull/3528/commits/0f5a455395c052840ab39ab9752e75103486313f
I think this is the cause of these changes.

Comment (rdig) [reply src/modules/core/components/InfoPopover/MemberInfoPopover/UserReputation.tsx:78]: Yeah, some rebase / merge screwed up your branch. We'll fix it tommorow.

**Comment by Julianahlk** [comment]
@chinins I implemented the changes you suggested and solved my merge conflicts with Raul yesterday. Can you please take a look at this PR again whenever you have a chance? Thank you!

**Review by chinins — APPROVED** [review]
All good - nice work!

**Comment by Julianahlk** [comment]
> All good - nice work!

Thank you!
